### PR TITLE
add simple docker pipeline for poc1

### PR DESCRIPTION
**Pull request by jacobbridges**
Simple pipeline which should _in theory_ perform the following actions:

- Authenticate with Google API via service account json file
- Authenticate Docker with Google Artifact Registry (GAR)
- Build the docker image in `poc1-api`
- Push the docker image to GAR, tagged with the commit hash

### CODE REVIEW

**Comment by jacobbridges** [comment]
Once I am confident this pipeline works for poc1, I will create similar pipelines for the other POCs.

File: .github/workflows/poc1-gar-pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          project_id: '${{ env.PROJECT_ID }}'
+
+      - name: 'Docker Auth'
+        run: |-
+          gcloud auth configure-docker ${{ env.GAR_LOCATION }}-docker.pkg.dev
+
+      - name: 'Docker Build'
+        working-directory: poc1-api
+        run: |-
+          docker build . --file Dockerfile --tag "${{ env.GAR_LOCATION }}-docker.pkg.dev/${{ env.PROJECT_ID }}/${{ env.REPOSITORY }}/${{ env.IMAGE }}:${{ github.sha }}"
```
Comment (IllyaMoskvin) [inline .github/workflows/poc1-gar-pipeline.yaml:44]: Does the `docker build` happen in `poc1-api/` or in root of repo? It should happen in subdir.

Comment (jacobbridges) [reply .github/workflows/poc1-gar-pipeline.yaml:44]: According to [the docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsworking-directory), the `working-directory` key on [line 42](https://github.com/IllyaMoskvin/looker-python/pull/1/commits/720c7112d7bd16766ba1682c8ae7287f85c42a68#diff-09fd6f94920907e0d590b50aa8de11ae352970d4b3b9fdc870e406d04a9f56ddR42) should cause this step to run within that directory.

Comment (IllyaMoskvin) [reply .github/workflows/poc1-gar-pipeline.yaml:44]: Nevermind, I just noticed the `working-directory` parameter. LGTM!

**Comment by Margaritagrinko** [comment]
great

**Pull request merged**